app/models/pydantic/query.py

```diff
@@ -11,9 +12,15 @@ class QueryRequestIn(StrictBaseModel):
 
 
 class QueryBatchRequestIn(StrictBaseModel):
-    feature_collection: Optional[FeatureCollection]
-    uri: Optional[str]
-    id_field: str
+    feature_collection: Optional[FeatureCollection] = Field(
+        None, description="An inline collection of GeoJson features on which to do the same query"
+    )
+    uri: Optional[str] = Field(
+        None, description="URI to a CSV or GeoJson file giving a list of features on which to do the same query.  For a CSV, the column with the geometry in WKB format should be named 'WKT'"
+    )
```

Comment:
I think technically many more file types are possible, no? It should be limited only by the preprocessing phase and the use of geopandas. I think it's written such that it will work with shapefiles, GPKGs, etc. Maybe you can write something vague (like "vector file") until we try all the possibilities?

Reply:
Also, is this accurate: "the column with the geometry in WKB format should be named 'WKT'"
That seems... wrong. WKT != WKB.

Reply:
OK, change the comments to refer to a vector file that is in any of a variety of formats supported by Geopandas, including GeoJson and CSV format.

Also, the column name does have to be WKT, even though the format is WKB..  I made that a bit clearer in the comments.